tools/deployment/make_linux_package.sh

```diff
@@ -154,18 +213,18 @@ function main() {
   cp $LENSES_LICENSE $INSTALL_PREFIX/$LENSES_DST
   cp $LENSES_SRC/*.aug $INSTALL_PREFIX/$LENSES_DST
 
-  if [[ $OSQUERY_CONFIG_SRC != "" ]] && [[ -f $OSQUERY_CONFIG_SRC ]]; then
-    log "config setup"
+  if [[ ! -z $OSQUERY_CONFIG_SRC ]] && [[ -f $OSQUERY_CONFIG_SRC ]]; then
```

Comment:
Check for existence of this and below files in the check_parsed_args function? We do this in the mac script.